res/qml/Button.qml

```diff
@@ -6,152 +6,162 @@ import "Theme"
-        }
-    }
-    contentItem: Item {
-        anchors.fill: parent
-
-        Glow {
-            id: labelGlow
-            anchors.fill: parent
-            color: label.color
-            radius: 1
-            source: label
-            spread: 0.1
-        }
-        Label {
-            id: label
-            anchors.fill: parent
-            color: root.normalColor
-            font.bold: true
-            font.capitalization: Font.AllUppercase
-            font.family: Theme.fontFamily
-            font.pixelSize: Theme.buttonFontPixelSize
-            horizontalAlignment: Text.AlignHCenter
-            text: root.text
-            verticalAlignment: Text.AlignVCenter
-            visible: root.text != null
-        }
-        Image {
-            id: image
-            anchors.centerIn: parent
-            asynchronous: true
-            fillMode: Image.PreserveAspectFit
-            height: icon.height
-            source: icon.source
-            visible: false
-            width: icon.width
-        }
-        ColorOverlay {
-            anchors.fill: image
-            antialiasing: true
-            color: root.normalColor
-            source: image
-            visible: icon.source != null
-        }
-    }
+    implicitHeight: 26
     states: [
         State {
             name: "pressed"
             when: root.pressed
 
             PropertyChanges {
-                color: root.checked ? Theme.accentColor : Theme.darkGray3
                 target: backgroundImage
+                color: root.checked ? "#3a60be" : "#3F3F3F"
             }
+
             PropertyChanges {
-                color: root.pressedColor
                 target: label
+                color: root.pressedColor
             }
+
             PropertyChanges {
                 target: labelGlow
                 visible: true
             }
+
         },
         State {
             name: "active"
             when: (root.highlight || root.checked) && !root.pressed
 
             PropertyChanges {
-                color: Theme.accentColor
                 target: backgroundImage
+                color: "#2D4EA1"
             }
+
             PropertyChanges {
-                color: root.activeColor
                 target: label
+                color: root.activeColor
             }
+
             PropertyChanges {
                 target: labelGlow
                 visible: true
             }
-            PropertyChanges {
-                color: Qt.darker(Theme.accentColor, 3)
-                target: bottomInnerEffect
-            }
-            PropertyChanges {
-                color: Qt.darker(Theme.accentColor, 3)
-                target: topInnerEffect
-            }
+
         },
         State {
             name: "inactive"
             when: !root.checked && !root.highlight && !root.pressed
 
+            // PropertyChanges {
+            //     target: backgroundImage
+            //     source: Theme.imgButton
+            // }
+
```

Comment:
This comment is meaningless, either describe why this code is commented, or remove it